Sources/Sentry/Public/SentryHttpStatusCodeRange.h

```diff
@@ -32,9 +32,6 @@ SENTRY_NO_INIT
  */
 - (instancetype)initWithStatusCode:(NSInteger)statusCode;
 
-// TODO: make it private
-- (BOOL)isInRange:(NSInteger)statusCode;
```

Comment:
Is it necessary to make this breaking change now or can it wait for 8?

Reply:
This is on top of my PR, not a problem.